README.md

```diff
@@ -29,10 +29,10 @@ To better communicate and more easily build an API and UI, we've decided on a co
 
 We are planning to use the following tools:
 
-- [Node.js](https://nodejs.org) / [Express](https://expressjs.com) for our backend
+- The [Node.js](https://nodejs.org) based [Next.js](https://nextjs.org) framework
```

Comment:
We're still technically using express, but I don't really know how to word it, maybe put next down with React and leave express here?

Reply:
We can write Node.js with Express in simple terms?

Reply:
@vaibhavsingh97 what do you think about
```
Node js with express
React js using next.js framework
```

Reply:
SGTM 👍